docs/sphinx/manual/creating_hpx_projects.rst

```diff
@@ -12,6 +12,144 @@
 Creating |hpx| projects
 =======================
 
+.. _makefile:
```

Comment:
Not sure if my comment got lost or if this will be a duplicate, but could you move this section to the bottom? And the note about using pkgconfig and CMake to the top of the makefile section? I think this is a very important section but we should make it very clear that we recommend against writing makefiles manually as it can be very error prone. Was this section motivated by the problems with hpx_wrap etc when using makefiles?

Reply:
@msimberg yes, dielph had trouble with building blazemark which was a makefile solution. He asked me to add documentation specifically for makefiles as well. Hence the section.